media-push/develop/integration-best-practices.mdx

```diff
@@ -185,12 +120,9 @@ Update API does not support the following configuration of the update `Converter
 
 ### Retry with retreat strategy
 
-If you receive `404`, `429`, or `5xx` error codes, take an escape strategy. For example, wait 5-6 seconds, 10-11 seconds,
-or 15-16 seconds and try again. If you receive `404` and `5xx` three times in a row or if the `state` field in the body of the
-response to querying the push status is `failed`, then the media push task has failed. Perform the following steps to rectify the fault:
+If you receive `404`, `429`, or `5xx` error codes, take an escape strategy. For example, wait 5-6 seconds, 10-11 seconds, or 15-16 seconds and try again. If you get three consecutive `404` or `5xx` error codes, or if the `state` field in the response to checking the status of a media push task says `failed`, it means that the media push task has failed. To fix this problem, follow these steps:
```

Comment:
I think backoff/escape/retreat strategy is the same thing - could we make sure to be consistent?

Reply:
Changed escape to backoff